packages/braid-design-system/package.json

```diff
@@ -216,7 +216,7 @@
     "@types/autosuggest-highlight": "^3.1.1",
     "@types/babel__core": "^7.20.1",
     "@types/lodash.curry": "^4.1.9",
-    "@types/node": "^18.19.68",
+    "@types/node": "^22.16.0",
```

Comment:
Had to update the node types so I could use the `glob` and `styleText` APIs. Also it's out of date relative to the configured node version anyway.